wynnset/tapestry-wp
# 595 - Tooltips at the Top of a Tapestry Get Hidden

Closes #595 
Generates tooltip location based on node coordiates

Steps to test: 
- Make a locked node
- Drag it to the top of the tapestry page
- Hover over it and make sure tooltip renders under node rather than on top
- Drag away and check that it resets
